ga4gh/client.py

```diff
@@ -16,12 +16,37 @@
 
 
 class HTTPClient(object):
-
-    workaroundGoogle = 'google'
-
     """
-    Simple HTTP client for the GA4GH protocol.
+    GA4GH Http Client
     """
+    workaroundGoogle = 'google'
+
+    class RunRequestContainer(object):
+        """
+        A container class for arguments to runRequest
+        """
+        def __init__(
+            self, request=None, url=None, protocolClass=None,
+                listAttr=None, requestMethod='post', id_=None):
```

Comment:
I think the term `request` is being heavily overloaded here to mean several different things. I'd suggest

- requestMethod -> httpMethod
- protocolClass -> protocolResponseClass
- request -> protocolRequest